DSharpPlus/Entities/DiscordWebhook.cs

```diff
@@ -107,7 +107,7 @@ public Task<DiscordMessage> ExecuteAsync(DiscordWebhookBuilder builder)
             => (this.Discord?.ApiClient ?? this.ApiClient).ExecuteWebhookAsync(this.Id, this.Token, builder.Content,
                 builder.Username.HasValue ? builder.Username.Value : this.Name, 
                 builder.AvatarUrl.HasValue ? builder.AvatarUrl.Value : this.AvatarUrl, 
-                builder.IsTTS, builder.Embeds, builder.Files, builder.Mentions);
+                builder.IsTTS, builder.Embeds, builder._files, builder.Mentions);
```

Comment:
Same here.

Reply:
This is actually probably gonna change anyway when i do my next pr which is a cleanup for the webhook builder and we are gonna pass the builder the whole way through. So if u want, we can wait for this request change till then

Reply:
After talking with @Neuheit in discord, this is gonna wait